code/datums/movement/mob.dm

```diff
@@ -1,24 +1,3 @@
-// Movement relayed to self handling
-/datum/movement_handler/mob/relayed_movement
```

Comment:
This appears to be a stub that Psi mentioned may date back to half-implemented borer move handling. It doesn't do anything and blocks relayed moves from mobs.